src/ontology/mondo-ingest.Makefile

```diff
@@ -274,13 +269,12 @@ $(REPORTDIR)/%_excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/%_excluded_terms_i
 	--outpath $@
 
 # Exclusions: all artefacts for single ontology
-exclusions-%:
-	$(MAKE) $(REPORTDIR)/$*_term_exclusions.txt
-	$(MAKE) $(REPORTDIR)/$*_exclusion_reasons.ttl
-	$(MAKE) $(REPORTDIR)/$*_excluded_terms_in_mondo_xrefs.tsv
+.PHONY: exclusions-%
+exclusions-%: reports/%_exclusion_reasons.ttl reports/%_excluded_terms_in_mondo_xrefs.tsv  $(REPORTDIR)/%_term_exclusions.txt 
+	echo "$@ completed"
```

Comment:
I undid a lot of the `$(MAKE)` pattern which I orginally introduced in good faith, not knowing that `-B` is passed into $(MAKE) as well.. which mean that a lot of goals were run multiple times for no reason.

Reply:
Oh, didn't realize that! I think I've seen that before. Good to know this.